app/controllers/file.js

```diff
@@ -57,8 +58,20 @@ FileController.prototype.downloadFile = function(opts) {
       .on("response", (res) => {
         this.hasDownloadError = false;
         if (res.statusCode == 200) {
-          this.writeFile(res);
-          this.emit("downloading");
+
+          let fileSize = res.headers["content-length"];
+          let self = this;
+          fileSystem.isThereAvailableSpace(this.logger, function(isThereAvailableSpace) {
```

Comment:
@rodrigopavezi Shouldn't this also be passing the `fileSize` value to `isThereAvailableSpace()`?